examples/getstarted/package.json

```diff
@@ -17,6 +17,7 @@
     "@strapi/plugin-documentation": "4.14.3",
     "@strapi/plugin-graphql": "4.14.3",
     "@strapi/plugin-i18n": "4.14.3",
+    "@strapi/plugin-releases": "4.14.3",
```

Comment:
ℹ️ I'm not sure if we want to install releases as it's own dep (similar to i18n) when we handle EE or if we should install it through @strapi/strapi like the other EE features.

Reply:
I think this is a product decision on whether it is optional or can not be uninstalled (core-plugin aka feature).

Reply:
> ℹ️ I'm not sure if we want to install releases as it's own dep (similar to i18n) when we handle EE or if we should install it through @strapi/strapi like the other EE features.

This needs discussing with product if we want to start doing EE plugins, with legal on the LICENSE requirements & also with engineering about any technical constraints we might meet.

I would do this before we merge this PR imo

Reply:
Based on both product & eng plans with v5 to make those core features packages using the plugin api & not plug&play plugins I would go with @strapi/releases (that name could be confusing though 🤔 )  & make this a dependency. 

It will then always be there ready to be activated if you add a license

Reply:
*this is somethign you can already do by using the same method we use to load e.g the CTB

Reply:
Ok sounds good. I'll move it. For the package name what are your thoughts on `@strapi/release-manager` 🤔

Reply:
maybe it's equally confusing 🤦‍♂️ : `@strapi/content-releases`..?

Reply:
The more the code and the wording we use to discuss / document the feature are the same the better. Ubiquitous language as they say :D 

We do add a concept of releases from what I saw so we can try to keep that around. content-releases sounds pretty decent. Go with your idea unless someone has a strong opinion argument against it honestly